webapp/titanembeds/templates/embed.html.j2

```diff
@@ -27,6 +27,11 @@
         {% if css.css is not none %}{{ css.css }}{% endif %}
       {% endif %}
     </style>
+    
+    <!--Provide default mobile metadata-->
+    <meta id=AppTitleApple name="apple-mobile-web-app-title" content="TitanEmbeds">
```

Comment:
`id="AppTitleApple"` (Use quotes around the value)

Reply:
Oh noooooooooooo